### ISSUE TITLE
Error: React is running in production mode, but dead code elimination has not been applied.

### ISSUE THREAD

**Issue by timohausmann** [theatre-js/theatre#346]
Hey, I followed your [r3f tutorial](https://www.theatrejs.com/docs/0.5/getting-started/with-react-three-fiber) on with vite react-ts and get this error: 

> Uncaught Error: React is running in production mode, but dead code elimination has not been applied. Read how to correctly configure React for production: https://reactjs.org/link/perf-use-production-build

Here is a demo (see console): https://vitejsvite9yegzh-cufs--5173.local-credentialless.webcontainer.io/
Editor: https://stackblitz.com/edit/vitejs-vite-9yegzh

Apart from the error everything seems to be working fine. Should I just catch the error?

**Comment by AriaMinaei** [comment]
Yeah this is very annoying but harmless for your app as it won't affect the final production bundle. The script that creates the bundle is [here](https://github.com/theatre-js/theatre/blob/main/packages/r3f/devEnv/bundle.ts) if anyone wants to contribute a patch.
